app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListActivity.kt

```diff
@@ -15,23 +16,32 @@ import com.github.epfl.meili.database.FirestoreDatabase
 import com.github.epfl.meili.home.Auth
 import com.github.epfl.meili.messages.ChatLogActivity
 import com.github.epfl.meili.models.Friend
+import com.github.epfl.meili.models.User
 import com.github.epfl.meili.profile.friends.NearbyActivity
+import com.github.epfl.meili.util.ClickListener
 import com.github.epfl.meili.util.MeiliViewModel
 import com.github.epfl.meili.util.TopSpacingItemDecoration
+import com.google.firebase.firestore.FirebaseFirestore
 
-class FriendsListActivity : AppCompatActivity() {
+//TODO: fix setting default image if none received e.g meili logo
```

Comment:
The TODOs in this activity will be in a later pr?

Reply:
It is done now! Thanks!